install/README.md

```diff
@@ -333,6 +339,46 @@ This is the current list of strings supported:
     - **Type:** `string`
     - **Default:** ``
 
+### CAS-specific environment variables
+
+    These variables specify the behaviour of CAS authentication. If `SS_CAS_AUTHENTICATION` is false,
+    none of the other ones are used.
```

Comment:
I'm just realizing we have this pattern of using code blocks to introduce sections in this document. I'm wondering if that was a pattern introduced accidentally. It doesn't make sense to me.

Reply:
I suspect you might be right, and maybe we never noticed because most of the time we're looking at the README through our code editors. I removed the code blocks in commit https://github.com/artefactual/archivematica-storage-service/pull/538/commits/5a70439dce21e800e748948e08800d8dcf253246.